peers/src/consensus_msg.rs

```diff
@@ -8,28 +8,17 @@ use mc_consensus_scp::Msg;
 use mc_crypto_digestible::{DigestTranscript, Digestible, MerlinTranscript};
 use mc_crypto_keys::{Ed25519Pair, Ed25519Signature, KeyError, SignatureError, Signer, Verifier};
 use mc_ledger_db::Ledger;
-use mc_transaction_core::{tx::TxHash, BlockID};
+use mc_transaction_core::{mint::MintConfigTx, tx::TxHash, BlockID};
 use serde::{Deserialize, Serialize};
 use std::{convert::TryFrom, hash::Hash, result::Result as StdResult};
 
 /// A single value in a consensus round.
 #[derive(
-    Clone,
-    Copy,
-    Display,
-    Debug,
-    PartialEq,
-    Eq,
-    PartialOrd,
-    Ord,
-    Hash,
-    Serialize,
-    Deserialize,
-    Digestible,
+    Clone, Debug, PartialEq, Eq, PartialOrd, Ord, Hash, Serialize, Deserialize, Digestible,
```

Comment:
The change here is that `Copy` and `Display` were removed - they weren't actually needed. `Copy` cannot be implemented for `MintConfigTx` because it contains a `Vec<>`, and `Display` is going to end up identical to `Debug`, so there's no point in having it.